### PR TITLE
fix typos and images in blog cn

### PR DESCRIPTION
**Pull request by husiyu**
@UNHNQ @YiniXu9506 PTAL

### CODE REVIEW

File: talk-tidb-pattern.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ tags: ['TiDB', '分布式数据库']
 
 它整个思路有点像 Bigtable，它相当于把底下的存储层分开，数据在最底层存储上已经没有表、行这样结构的划分，每一块数据都由一个固定的 size 比如 64 M、128 M 连续的 Key-value pairs 组成。其实这个模型背后最早的系统应该是谷歌在 06 年发表的 Bigtable 这篇论文里面去描述的。这个模型有什么好处呢？一是它能真正实现这种弹性的扩展。第二个，它是一个真正高度去中心化。去中心化这个事情，对于一个大的 Cluster 来说是一个非常重要的特性。
 
-**还有一个优势，在 KV 层实现真正具有一定的自动 Failover 的能力。 **Failover指的是什么呢？比如说在一个集群比较大的情况下，或者你是一个 cluster ，你任何一个节点，任何一个数据损坏，如果能做到业务端的透明，你就真正实现了 Auto-Failover 的能力。其实在一些对一致性要求不那么高的业务里面，Auto-Failover 就是指， 比如在最简单的一个 MySQL 组从的模型里，当你的组挂掉了以后，我监控的程序自动把 slave  提上来，这也是一种 Failover 的方式。但是这个一致性或者说数据的正确性并不能做到很好的保证。你怎么做到一致性的 Auto-Failover，其实背后需要做非常非常多的工作。
+**还有一个优势，在 KV 层实现真正具有一定的自动 Failover 的能力。**Failover指的是什么呢？比如说在一个集群比较大的情况下，或者你是一个 cluster ，你任何一个节点，任何一个数据损坏，如果能做到业务端的透明，你就真正实现了 Auto-Failover 的能力。其实在一些对一致性要求不那么高的业务里面，Auto-Failover 就是指， 比如在最简单的一个 MySQL 组从的模型里，当你的组挂掉了以后，我监控的程序自动把 slave  提上来，这也是一种 Failover 的方式。但是这个一致性或者说数据的正确性并不能做到很好的保证。你怎么做到一致性的 Auto-Failover，其实背后需要做非常非常多的工作。
```
Comment (YiniXu9506) [inline talk-tidb-pattern.md]: Please don't bold the punctuation in this article

File: the-design-and-implementation-of-multi-raft.md [hunk 2]
```diff
@@ -202,13 +25,13 @@ PD 是一个非常重要的中心节点，它通过集成 etcd，自动的支持
 
 PD 的 Client trait 非常简单，多数都是对集群元信息的 set/get 操作，需要额外注意的几个：
 
-bootstrap\_cluster：当我们启动一个 TiKV 服务的时候，首先需要通过 is\_cluster\_bootstrapped 来判断整个 TiKV 集群是否已经初始化，如果还没有初始化，我们就会在该 TiKV 服务上面创建第一个 region。
+**bootstrap\_cluster：**当我们启动一个 TiKV 服务的时候，首先需要通过 is\_cluster\_bootstrapped 来判断整个 TiKV 集群是否已经初始化，如果还没有初始化，我们就会在该 TiKV 服务上面创建第一个 region。
```
Comment (YiniXu9506) [inline the-design-and-implementation-of-multi-raft.md]: Please do not bold the punctuation in this article

**Comment by YiniXu9506** [comment]
LGTM

**Comment by husiyu** [comment]
LGTM

**Pull request merged**